x-pack/plugins/cloud_security_posture/public/pages/findings/findings.tsx

```diff
@@ -59,6 +59,13 @@ export const Findings = () => {
                   display: block;
                 `}
                 label="Beta"
+                tooltipContent={
+                  <FormattedMessage
+                    id="xpack.csp.findings.betaLabel"
```

Comment:
@Omolola-Akinleye can you double check the id is according to the i18n standards

Reply:
@kfirpeled  This Id correctly adheres to i18n standard and passes the translations check.